MetaMask/metamask-extension
# Updating account and network icons

## Explanation

Currently, we are using image tags and svg imports for a icons inside of the network and account menus.

This is a problem because these icons will not work for theme compatibility and dark mode.

In order to solve this problem, this pull request updates all image icons for the account network menu and updates them with icon components that accept a color prop among others.

## More information
- Replaced all svg image tags with corresponding icon components
- Removed unused svg files
- Created consistent Icon components that follow the same component api, accept css variables and also inherit the font color if inside of a button or next to text.
- Removed unused SCSS

## Screenshots/Screencaps

### Before

<img width="350" src="https://user-images.githubusercontent.com/8112138/158286883-3f6f4110-d931-4e6e-9728-39a399ec56c5.png" /><img width="350" src="https://user-images.githubusercontent.com/8112138/158286884-0ce7b474-9627-45bd-a693-4c3dd0bb19f6.png" />


### After

<img width="367" alt="Screen Shot 2022-03-14 at 6 17 44 PM" src="https://user-images.githubusercontent.com/8112138/158286966-4ef49196-9072-4298-bc2d-a92cd99d6ea6.png"><img width="339" alt="Screen Shot 2022-03-14 at 6 17 58 PM" src="https://user-images.githubusercontent.com/8112138/158286968-eee85695-600b-4dae-938a-e0366ba7ab84.png">

### Storybook updates

<img width="1440" alt="Screen Shot 2022-03-14 at 3 03 02 PM" src="https://user-images.githubusercontent.com/8112138/158287001-3bf397d9-a949-40b2-9ab7-7ea4991a9f77.png">

![screencapture-localhost-6006-2022-03-14-15_22_38](https://user-images.githubusercontent.com/8112138/158287003-4a3e94c0-5f3e-43e1-ab5e-246b6841568e.png)

## Manual testing steps

- Go to account menu
- See icon changes 
- Go to network menu
- See checkmark icon changes

Storybook Icon Stories and Docs
- Go to the latest build of storybook in this PR
- Search `Icon` go to the second result
- See new stories and documentation updates
